logstash-core/src/main/java/org/logstash/Valuefier.java

```diff
@@ -44,16 +42,16 @@ private static Object convertJavaProxy(JavaProxy jp) {
         }
     }
 
-    public static Object convertNonCollection(Object o) {
+    private static Object convertNonCollection(Object o) {
         try {
-            return o == null ? BiValues.NULL_BI_VALUE : BiValues.newBiValue(o);
+            return BiValues.newBiValue(o);
```

Comment:
@jakelandis yea (but I think we could remove that check as well), the important part in this PR is that we'll never see a `null` in this method and hence that check can be removed from the hot path :)